src/main/kotlin/app/hashers/FactHasher.kt

```diff
@@ -105,4 +145,46 @@ class FactHasher(private val serverRepo: Repo = Repo(),
             Logger.info { "Sent ${facts.size} facts to server" }
         }
     }
+
+    private fun calcIncAvg(prev: Double, element: Double, count: Long):
+        Double {
```

Comment:
btw, I find the style when arguments are on new lines more readable than putting a return type on a new line